pkg/engine/engine.go

```diff
@@ -47,7 +47,25 @@ func (e Engine) Render(templates map[string][]byte, values map[string]interface{
 
 		// TODO: Could we detect unused variables, and give warning about those?
 
-		rendered[name] = []byte(output)
+		// File names can be templates to, render them here as well
+
+		buf.Reset()
```

Comment:
Dunno if it makes sense to reuse or just create another one

Reply:
I think creating new Builder is fine. But then the `buf.Reset()` is redundant here, since the garbage collector will handle it anyway.